Userland/Libraries/LibGfx/JPEGLoader.cpp

```diff
@@ -313,7 +313,13 @@ static inline i32* get_component(Macroblock& block, unsigned component)
 
 static ErrorOr<void> add_dc(JPEGLoadingContext& context, Macroblock& macroblock, ScanComponent const& scan_component)
 {
-    auto& dc_table = context.dc_tables.find(scan_component.dc_destination_id)->value;
+    auto maybe_table = context.dc_tables.get(scan_component.dc_destination_id);
+    if (!maybe_table.has_value()) {
+        dbgln_if(JPEG_DEBUG, "Unable to find a DC table with id: {}", scan_component.dc_destination_id);
```

Comment:
Is there much value in logging this? You'll see the actual error right after anyways, right? (Up to you.)

Reply:
The message gives you the table ID and the error doesn't.
That's not super-duper useful, but it won't hurt either